shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -228,6 +228,16 @@ TEST(EmbedderTestNoFixture, CanGetCurrentTimeInNanoseconds) {
   ASSERT_LT((point2 - point1), fml::TimeDelta::FromMilliseconds(1));
 }
 
+TEST_F(EmbedderTest, CanReloadSystemFonts) {
```

Comment:
This test make sure the hook from embedding -> embedding_engine -> shell works. It does not test for functionality.